test/functional/ui/incsubstitution_spec.lua

```diff
@@ -0,0 +1,1214 @@
+local helpers = require('test.functional.helpers')(after_each)
+local Screen = require('test.functional.ui.screen')
+local clear = helpers.clear
+local curbufmeths = helpers.curbufmeths
+local eq = helpers.eq
+local eval = helpers.eval
+local execute = helpers.execute
+local expect = helpers.expect
+local feed = helpers.feed
+local insert = helpers.insert
+local meths = helpers.meths
+local neq = helpers.neq
+local source = helpers.source
+
+local default_text = [[
+  Inc substitution on
+  two lines
+]]
+
+local function common_setup(screen, incsub, text)
+  if screen then
+    execute("syntax on")
+    execute("set nohlsearch")
+    execute("hi IncSubstitute guifg=red guibg=yellow")
+    screen:attach()
+    screen:set_default_attr_ignore( {{bold=true, foreground=Screen.colors.Blue}} )
```

Comment:
please remove this, by the new standard #5197  we prefer to not ignore anything, unless there's a really good reason (and thanks to `snapshot_util()` there typically isn't)